hibernate-core/src/main/java/org/hibernate/type/TypeHelper.java

```diff
@@ -337,7 +336,6 @@ else if ( !types[i].isAssociationType() ) {
 			final Object[] previousState,
 			final boolean[][] includeColumns,
 			final boolean[] includeProperties,
-			final boolean anyUninitializedProperties,
 			final SharedSessionContractImplementor session) {
```

Comment:
@sebersole, is TypeHelper#findModified considered an SPI? Would removing anyUninitializedProperties argument break applications?

Is it worth deprecating this method and adding a new method that excludes anyUninitializedProperties?